Dangerfile

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Make sure non-trivial amounts of code changes come with corresponding tests
+has_app_changes = !git.modified_files.grep(/lib/).empty? || !git.modified_files.grep(/app/).empty?
+has_test_changes = !git.modified_files.grep(/test/).empty?
```

Comment:
should this be `/spec/` instead of `/test/`?

Reply:
ops thank you for catching it! I'll update